packages/snap-preact-components/src/components/Atoms/SearchHeader/SearchHeader.tsx

```diff
@@ -18,24 +18,24 @@ const CSS = {
 export const SearchHeader = observer((properties: SearchHeaderProps): JSX.Element => {
 	const globalTheme: Theme = useTheme();
 
-	const landingPage = properties.controller?.store.merchandising.landingPage;
+	const landingPage = properties.controller?.store.merchandising.landingPage || properties.merchandisingStore?.landingPage;
 
 	const pagination = properties.controller?.store.pagination || properties.paginationStore;
 	const search = properties.controller?.store.search || properties.queryStore;
 
 	const defaultProps: Partial<SearchHeaderProps> = {
 		titleText: `Showing ${
-			pagination?.multiplePages ? `<span class="ss-results-count-range"> ${pagination?.begin} - ${pagination?.end} of </span>` : ''
+			pagination?.multiplePages ? `<span class="ss__search-header__results-count-range"> ${pagination?.begin} - ${pagination?.end} of </span>` : ''
 		} 
-		<span class="ss-results-count-total">${pagination?.totalResults}</span> 
+		<span class="ss__search-header__results-count-total">${pagination?.totalResults}</span> 
 		result${pagination?.totalResults == 1 ? '' : 's'} 
-		${search?.query ? `<span>for <span class="ss-results-query">"${search.query.string}"</span></span>` : ''}
+		${search?.query ? `<span>for <span class="ss__search-header__results-query">"${search.query.string}"</span></span>` : ''}
 	`,
-		oqText: `<div class="ss-oq">No results found for <em>"${search?.originalQuery?.string}"</em>, showing results for <em>"${search?.query?.string}"</em> instead.</div>`,
+		correctedQueryText: `<div class="ss__search-header__corrected">No results found for <em>"${search?.originalQuery?.string}"</em>, showing results for <em>"${search?.query?.string}"</em> instead.</div>`,
```

Comment:
Is this wrapping div necessary since the component wraps this already with `ss__search-header__corrected`?